app/src/main/java/com/pec_acm/moviedroid/ProfileActivity.kt

```diff
@@ -29,6 +35,46 @@ class ProfileActivity : AppCompatActivity() {
             .load(user.photoUrl)
             .placeholder(R.drawable.ic_baseline_account_circle_24)
             .into(binding.imgProfile)
+
+            var movieRate = 0
+            var tvRate = 0
+            var movieCount = 0
+            var tvCount = 0
+
+            userReference.child(user.uid).get().addOnCompleteListener {
+                val userDbRef = it.result.getValue(User::class.java)
+                for (item in userDbRef!!.userList)
+                {
+                    if (item.personalScore != 0)
+                    {
+                        if (item.category == "tv")
+                        {
+                            tvCount += 1
+                            tvRate += item.personalScore
+                        }
+                        else
+                        {
+                            movieCount += 1
+                            movieRate += item.personalScore
+                        }
+                    }
```

Comment:
The logic of getting user info should be inside a function in a view model, create a `ProfileViewModel` and add this logic there and observe a live data of the user to then calculate and display the ratings